frontend/src/app/commonComponents/Header.tsx

```diff
@@ -184,7 +184,7 @@ const Header = () => {
                   color: "white",
                 }}
               >
-                Test Queue
+                Conduct Test
```

Comment:
Seems￼￼ like a constant would be good here, but that may be more of a "when we internationalize" concern.